plugins/woocommerce/changelog/fix-43608-products-table-headers-scope

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Add scope attribute to the product attributes table
```

Comment:
maybe also add `aria-label` you added as part of this PR.

Reply:
@amesplant please add the above to the changelog.

Reply:
@naman03malhotra cc @vikrampm1 updated changelog